latest/index.bs

```diff
@@ -205,6 +205,37 @@ Metadata {#metadata}
 The various `.zattrs` files throughout the above array hierarchy may contain metadata
 keys as specified below for discovering certain types of data, especially images.
 
+"axes" metadata {#axes-md}
+--------------------------
+
+"axes" describes the dimensions of a physical coordinate space. It is a list of dictionaries, where each dictionary describes an dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
+- SHOULD contain the field "type" to specify the type of this dimension. The value SHOULD be one of "space", "channel" or "time".
+- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be a valid unit according to UDUNITS-2.
```

Comment:
> does this mean we cannot have spatial order `x`,`y`,`z` or if i wanted to call this `i`, `j`, `k` to distinguish from physical space xyz as the mapping by transforms that would not be possible. at this stage i would rather get some metadata in than not, so consider these clarifications being sought.

Sorry, I don't understand this question. For some context: `name` has arbitrary values now, so we cannot impose an order based on it, but instead impose the order based on `type`. I hope this clarifies things; if not please rephrase the question.


> and `must be up to` suggests that any of these components are optional. doesn't this mean that you would still require names to say do `tyx` or `czyx` ?

Again, this is specified by `type` now and we require 2 or 3 axis of spatial type and one optional channel and one optional time axis.

Reply:
I still wonder if (or where) we should still state that the convention for ordering spatial axes is `zyx`?
Some viewers (like vizarr) are very much 2D viewers (with Z-slider). And our scaling (in ome-zarr-py) for multiscales is currently operating on XY only (not Z). napari can do orthoganal views, but *currently* it expects the last 2 axes to be `yx`.
So, support for other axes orders is less good and needs work. If you create an image with `xyz` or `xzy` axes, you'll find that it's not handled as well. This may come as a surprise to users unless it's either stated in the spec, or we have some other way of documenting the limitations of various clients/tools?

Reply:
@constantinpape - this question arose from @will-moore 's comment referring to this sentence

> All image arrays must be up to 5-dimensional with dimension order (t, c, z, y, x).

here it seems to connect dimension with name. if names are not important and only useful for connecting to visualization and interpretation, perhaps this can be restated as since zyx do not mean anything:

`All image arrays must be up to 5-dimensional with dimension order of type (time, channel, spatial, spatial, spatial)`.

i do like the current formulation of disconnecting from names for representation. if the future goal would be to relax the order, perhaps one could add that this restriction could be removed in future versions of the spec.

Reply:
@will-moore I see your point. I will think about a formulation on recommended axis order for axes that map to `zyx` and tag you there so we can discuss this.

@satra 

> All image arrays must be up to 5-dimensional with dimension order (t, c, z, y, x).

Yes, this part of the spec is outdated and I have not updated it yet because it is in the "unstructured part" of the spec and I wanted to get things right in the other part first.
I will update it now.

Reply:
```suggestion
- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be one of the following strings, which are valid units according to UDUNITS-2.
- Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
- Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
```